mffpy/tests/test_writer.py

```diff
@@ -110,3 +111,58 @@ def test_writer_exports_JSON():
         remove(filename)
     except BaseException:
         raise AssertionError(f"""Clean-up failed of '{filename}'.""")
+
+##START##
```

Comment:
Start streaming bin writer testing?

Reply:
I added that to denote where I started adding tests and shortly thereafter stopped seeing it at all. I will delete it.